packages/cli/test/integration/shared/utils/testServer.ts

```diff
@@ -198,7 +199,13 @@ export const setupTestServer = ({
 						registerController(
 							app,
 							config,
-							new AuthController({ config, logger, internalHooks, repositories }),
+							new AuthController({
+								config,
+								logger,
+								internalHooks,
+								repositories,
+								userService: Container.get(UserService),
```

Comment:
I think we should switch away from this pattern, and start using `this.userService = Container.get(UserService)` in the `Controller` constructor. That way, we don't have to keep maintaining  dependency list in two places.